mantidproject/mantid
# Disable show instrument if there is no spectra axis

**Description of work.**

see title

**To test:**

1. Load some data such as MAR11060
1. right click on the workspace. Show Instrument should be enabled
1. Run ConvertSpecturmAxis to create a new ws Theta, converting the sp axis to Theta.
1. Right click on the theta ws, the show instrument option should be disabled.

Fixes #24126.


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
